src/BrefKernel.php

```diff
@@ -122,6 +123,24 @@ protected function prepareCacheDir(string $readOnlyDir, string $writeDir): void
         ));
     }
 
+    /**
+     * Even though applications should never write into it on Lambda, there are parts of Symfony
+     * (like "about" CLI command) that expect the log dir exists, so we have to make sure of it.
+     *
+     * @see https://github.com/brefphp/symfony-bridge/issues/42
+     */
+    protected function ensureLogDir(string $writeLogDir): void
```

Comment:
```suggestion
    private function ensureLogDir(string $writeLogDir): void
```

If we make it protected, then it's part of the API of this package and we can't have BC break on this. I'd rather not expose anything we don't have to.

Reply:
I went with the same visibility as `prepareCacheDir()` has, but yeah, makes sense.